utils/rpms/daos.spec

```diff
@@ -142,6 +142,7 @@ Summary: The DAOS test suite
 Requires: %{name}-client = %{version}-%{release}
 Requires: python-pathlib
 Requires: fio
+Requires: json-c
```

Comment:
I think you need something in the spec file for Build..ask Brian if that's needed or if it will figure it out automatically.